config/initializers/assets.rb

```diff
@@ -5,3 +5,8 @@
 
 # Add additional assets to the asset load path
 Rails.application.config.assets.paths << Rails.root.join('node_modules')
+
+# Fix sassc sometimes segfaulting
+Rails.application.config.assets.configure do |env|
+  env.export_concurrent = false
+end
```

Comment:
!!!!!!!!! OMG

Reply:
hopefully works? 🤷🏼 

Tried updating a bunch of dependencies to see if there was some weird Ruby 3 interaction with a library, but was getting constantly getting:

```
Randomized with seed 50006
.........................../home/circleci/identity-idp/vendor/bundle/ruby/3.0.0/gems/sassc-2.4.0/lib/sassc/engine.rb:43: [BUG] Segmentation fault at 0x0000000000000000
ruby 3.0.3p157 (2021-11-24 revision 3fb7d2cadc) [x86_64-linux]
```

Happened on 3.0.2 and 3.0.3, always on `rack_attack_spec.rb` and finally gave up and took a suggestion from https://github.com/rails/sprockets/issues/633

Reply:
I hit that segfault very often in local development I would love to see this land on `main` sooner

Reply:
Unfortunately didn't work :|

Reply:
There are also mentions of having the issue if both `Rails.application.config.assets.precompile` and `app/assets/config/manifest.js` are used, but we don't do that.

The other popular option is downgrading to sprockets `3.7.2`, but that doesn't work for us since the 4.x line includes the keyword arguments fixes